pkg/scheduler/scheduler_test.go

```diff
@@ -1200,3 +1197,122 @@ func TestSchedulerBinding(t *testing.T) {
 		})
 	}
 }
+
+// TestInjectingPluginConfigForVolumeBinding tests injecting
+// KubeSchedulerConfiguration.BindTimeoutSeconds as args for VolumeBinding if
+// no plugin args is configured for it.
+// TODO remove when KubeSchedulerConfiguration.BindTimeoutSeconds is eliminated
+func TestInjectingPluginConfigForVolumeBinding(t *testing.T) {
+	policyAlgorithmSC := schedulerapi.SchedulerAlgorithmSource{
+		Policy: &config.SchedulerPolicySource{},
+	}
+
+	defaultPluginConfigs := []config.PluginConfig{
+		{
+			Name: "VolumeBinding",
+			Args: &config.VolumeBindingArgs{
+				BindTimeoutSeconds: 600,
+			},
+		},
+	}
+
+	tests := []struct {
+		name             string
+		opts             []Option
+		wantPluginConfig []config.PluginConfig
+	}{
+		{
+			name:             "default with provider",
+			wantPluginConfig: defaultPluginConfigs,
+		},
+		{
+			name: "default with policy",
+			opts: []Option{
+				WithAlgorithmSource(policyAlgorithmSC),
+			},
+			wantPluginConfig: defaultPluginConfigs,
+		},
+		{
+			name: "customize BindTimeoutSeconds with provider",
+			opts: []Option{
+				WithBindTimeoutSeconds(100),
+			},
+			wantPluginConfig: []config.PluginConfig{
+				{
+					Name: "VolumeBinding",
+					Args: &config.VolumeBindingArgs{
+						BindTimeoutSeconds: 100,
+					},
+				},
+			},
+		},
+		{
+			name: "customize BindTimeoutSeconds with policy",
+			opts: []Option{
+				WithAlgorithmSource(policyAlgorithmSC),
+				WithBindTimeoutSeconds(100),
```

Comment:
in the case of policy, this comes from command line flags, correct?

Reply:
yes